### ISSUE TITLE
Error publishing resume – TypeError: Cannot read property 'username' of undefined

### ISSUE THREAD

**Issue by JulianNorton** [jsonresume/resume-cli#137]
Not sure what's going on, I'm trying to `resume register`

```There has been an error publishing your resume```

```/usr/local/lib/node_modules/resume-cli/lib/register/index.js:62
                    username: res.username,
                                 ^
TypeError: Cannot read property 'username' of undefined
    at /usr/local/lib/node_modules/resume-cli/lib/register/index.js:62:34
    at /usr/local/lib/node_modules/resume-cli/lib/register/register-user.js:23:17
    at Request.callback (/usr/local/lib/node_modules/resume-cli/node_modules/superagent/lib/node/index.js:781:12)
    at ClientRequest.<anonymous> (/usr/local/lib/node_modules/resume-cli/node_modules/superagent/lib/node/index.js:736:10)
    at ClientRequest.EventEmitter.emit (events.js:95:17)
    at Socket.socketErrorListener (http.js:1547:9)
    at Socket.EventEmitter.emit (events.js:95:17)
    at net.js:833:16
    at process._tickCallback (node.js:415:13)```

**Comment by matthewcocco** [comment]
+1

**Comment by thomasdavis** [comment]
Sorry guys, that is fixed in version 0.4.8
